sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -334,13 +394,14 @@ async def create_database_if_not_exists(  # pylint: disable=redefined-builtin
         :param str id: ID (name) of the database to read or create.
         :keyword offer_throughput: The provisioned throughput for this offer.
         :paramtype offer_throughput: Union[int, ~azure.cosmos.ThroughputProperties]
+        :param return_headers: Specifies function to return either a DatabaserProxy or a CosmosDict instance.
         :keyword dict[str, str] initial_headers: Initial headers to be sent as part of the request.
         :keyword response_hook: A callable invoked with the response metadata.
         :keyword int throughput_bucket: The desired throughput bucket for the client
         :paramtype response_hook: Callable[[Dict[str, str], Dict[str, Any]], None]
         :raises ~azure.cosmos.exceptions.CosmosHttpResponseError: The database read or creation failed.
-        :returns: A DatabaseProxy instance representing the database.
-        :rtype: ~azure.cosmos.DatabaseProxy
+        :returns: A DatabaseProxy instance representing the database or a CosmosDict with the response headers
+        :rtype: ~azure.cosmos.DatabaseProxy or CosmosDict
```

Comment:
```suggestion
        :rtype: ~azure.cosmos.DatabaseProxy or ~azure.cosmos.CosmosDict
```
Without the full reference, you wont be able to link through in the reference documentation